### ISSUE TITLE
Installation Problem on Laravel 11

### ISSUE THREAD

**Issue by 3x1io** [vormkracht10/laravel-seo-scanner#68]
When attempting to install the package on the latest version of Laravel 11, there is a problem unless I use the `-w` flag during installation. Without this flag, the installation fails. Can you please update the package to officially support Laravel 11 by bumping the required versions?

## Steps to reproduce:

1. Try to install the package on Laravel 11.
2. Notice the issue unless using the -w flag.

## Expected behavior:

The package should install seamlessly without the need for the -w flag on Laravel 11.

## Additional context:

• Working fine on Laravel 10 without any extra flags.

I would appreciate it if the dependencies could be updated to support Laravel 11 officially!

**Comment by Baspa** [comment]
I just installed it in a Laravel 11 project without any issues.. @3x1io 


<img width="761" alt="Screenshot 2024-11-12 at 11 58 38" src="https://github.com/user-attachments/assets/57e30aae-a2ce-4744-8203-e7e3bd417c12">

Can you provide me a reproduction repo?

**Comment by Baspa** [comment]
Dependencies seems to not be an issue.

**Comment by plompd** [comment]
I have the same issue on a Laravel 11.x project. Even the `-W` option doesn't work for me.


```
 - vormkracht10/laravel-seo-scanner[v1.0.0, ..., v1.1.1] require illuminate/contracts ^9.0 -> found illuminate/contracts[v9.0.0, ..., v9.52.16] but these were not loaded, likely because it conflicts with another require.
    - vormkracht10/laravel-seo-scanner[v1.2.0, ..., v1.3.7, v2.0.0, ..., v2.6.1, v3.0.0, ..., v3.10.0] require illuminate/contracts ^9.0|^10.0 -> found illuminate/contracts[v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.24] but these were not loaded, likely because it conflicts with another require.
    - vormkracht10/laravel-seo-scanner[v3.11.0, ..., v3.12.3] require symfony/dom-crawler ^6.2 -> found symfony/dom-crawler[v6.2.0, ..., v6.4.13] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires vormkracht10/laravel-seo-scanner * -> satisfiable by vormkracht10/laravel-seo-scanner[v1.0.0, ..., v1.3.7, v2.0.0, ..., v2.6.1, v3.0.0, ..., v3.12.3].
```

**Comment by Baspa** [comment]
Can you provide me an example of the `composer.json`? @plompd

**Comment by plompd** [comment]
```
"php": "8.3.*",
"ext-dom": "*",
"ext-simplexml": "*",
"ext-xmlwriter": "*",
"algolia/algoliasearch-client-php": "3.*",
"algolia/scout-extended": "3.*",
"archilex/filament-filter-sets": "3.*",
"artesaos/seotools": "1.3.*",
"awcodes/filament-curator": "*",
"bezhansalleh/filament-shield": "3.*",
"cheesegrits/filament-google-maps": "3.*",
"coconutcraig/laravel-postmark": "3.*",
"devlogx/filament-pirsch-dashboard-widget": "^1.1",
"diogogpinto/filament-auth-ui-enhancer": "^1.0",
"doctrine/dbal": "3.*",
"esign/laravel-redirects": "^2.0",
"fakerphp/faker": "1.*",
"filament/filament": "3.2.115",
"filament/spatie-laravel-settings-plugin": "3.*",
"filament/spatie-laravel-tags-plugin": "3.*",
"guzzlehttp/guzzle": "7.8.*",
"hammerstone/fast-paginate": "^1.1",
"hidehalo/nanoid-php": "1.*",
"icetalker/filament-table-repeater": "1.*",
"joshembling/image-optimizer": "1.*",
"koossaayy/laravel-mapbox": "0.*",
"laravel/framework": "11.*",
"laravel/horizon": "5.*",
"laravel/jetstream": "5.*",
"laravel/pulse": "1.*",
"laravel/sanctum": "4.*",
"laravel/scout": "10.11.5",
"laravel/telescope": "5.*",
"laravel/tinker": "2.*",
"livewire/livewire": "3.*",
"livewire/volt": "1.*",
"lorisleiva/laravel-actions": "2.*",
"lukeraymonddowning/self-healing-urls": "0.*",
"mcamara/laravel-localization": "^2.0",
"mews/purifier": "3.*",
"mohamedsabil83/filament-forms-tinyeditor": "2.*",
"openai-php/laravel": "*",
"orangehill/iseed": "3.*",
"owen-it/laravel-auditing": "13.*",
"panphp/pan": "^0.1.3",
"predis/predis": "2.*",
"pxlrbt/filament-environment-indicator": "2.*",
"pxlrbt/filament-excel": "2.*",
"ralphjsmit/laravel-filament-activitylog": "1.*",
"ralphjsmit/laravel-filament-record-finder": "1.*",
"roach-php/core": "3.*",
"roach-php/laravel": "3.*",
"ryangjchandler/laravel-cloudflare-turnstile": "1.*",
"saade/filament-laravel-log": "3.*",
"saloonphp/xml-wrangler": "1.*",
"spatie/browsershot": "4.*",
"spatie/laravel-activitylog": "4.*",
"spatie/laravel-feed": "4.*",
"spatie/laravel-ignition": "2.*",
"spatie/laravel-mailcoach-mailer": "1.*",
"spatie/laravel-mailcoach-sdk": "1.*",
"spatie/laravel-permission": "6.*",
"spatie/laravel-ray": "1.*",
"spatie/laravel-sitemap": "7.*",
"spatie/laravel-sluggable": "3.*",
"spatie/laravel-tags": "4.*",
"spatie/schema-org": "3.*",
"stevebauman/purify": "6.*",
"webbingbrasil/filament-advancedfilter": "3.*",
"ysfkaya/filament-phone-input": "2.*"
```

Any other details?

**Comment by Baspa** [comment]
Fixed this in v3.12.4, can you guys check if this works for you @plompd / @3x1io
